src/server/server.ts

```diff
@@ -1,18 +1,8 @@
 import * as WebSocket from 'ws';
 
-import {
-  SUBSCRIPTION_FAIL,
-  SUBSCRIPTION_DATA,
-  SUBSCRIPTION_START,
-  SUBSCRIPTION_END,
-  SUBSCRIPTION_SUCCESS,
-  KEEPALIVE,
-  INIT,
-  INIT_FAIL,
-  INIT_SUCCESS,
-} from './messageTypes';
-import {GRAPHQL_SUBSCRIPTIONS} from './protocols';
-import {SubscriptionManager} from 'graphql-subscriptions';
+import MessageTypes from '../common/message-types';
+import { GRAPHQL_WS } from '../common/protocol';
+import { SubscriptionManager } from 'graphql-subscriptions';
```

Comment:
Hey I completely agree with you here! We should define an interface definition for the subscriptions manager, so anyone that wants to use our transport package but only the client part knows exactly what it should be implemented in order to all work as expected :)

Good stuff!